activerecord/lib/active_record/internal_metadata.rb

```diff
@@ -38,10 +34,8 @@ def table_exists?
       def create_table
         unless table_exists?
           connection.create_table(table_name, id: false) do |t|
-            t.column :key,   :string, null: false, limit: KEY_LIMIT
-            t.column :value, :string
-            t.index  :key, unique: true, name: index_name
-
+            t.string :key, primary_key: true, limit: KEY_LIMIT
```

Comment:
Yes in `InternalMetadata`, but we need to recreate `active_record_internal_metadatas` table.
No in `SchemaMigration`, existing applications already created `schema_migrations` table.